mono/metadata/object.c

```diff
@@ -4505,6 +4512,8 @@ free_main_args (void)
 int
 mono_runtime_set_main_args (int argc, char* argv[])
 {
+	MONO_ENTER_GC_UNSAFE;
+
 	MONO_REQ_GC_NEUTRAL_MODE;
```

Comment:
Doesn't make sense to require GC Neutral mode immediately after switching to GC Unsafe.

But also this function doesn't need to be in GC Unsafe - it only interacts with eglib stuff.

Reply:
Oh, the second part. So undo this? I guess I'll look more closely, instead of just following "external only" marks, and replace with a comment.

Reply:
I didn't get around to this either.